editors/code/src/net.ts

```diff
@@ -68,32 +66,31 @@ interface DownloadOpts {
 }
 
 export async function download(opts: DownloadOpts) {
-    // Put the artifact into a temporary folder to prevent partially downloaded files when user kills vscode
-    await withTempDir(async tempDir => {
-        const tempFile = path.join(tempDir, path.basename(opts.dest));
-
-        await vscode.window.withProgress(
-            {
-                location: vscode.ProgressLocation.Notification,
-                cancellable: false,
-                title: opts.progressTitle
-            },
-            async (progress, _cancellationToken) => {
-                let lastPercentage = 0;
-                await downloadFile(opts.url, tempFile, opts.mode, (readBytes, totalBytes) => {
-                    const newPercentage = (readBytes / totalBytes) * 100;
-                    progress.report({
-                        message: newPercentage.toFixed(0) + "%",
-                        increment: newPercentage - lastPercentage
-                    });
-
-                    lastPercentage = newPercentage;
+    // Put artifact into a temporary file (in the same dir for simplicity)
+    // to prevent partially downloaded files when user kills vscode
+    const tempFile = `${opts.dest}.tmp`;
```

Comment:
Should we perhaps add a random suffix as well? Just to make sure that there's no sudden interfernce somewhere. 

Otherwise, bors d+

Reply:
@lnicola what do you think, having the fixed filename will prevent us from creating unlimited amount of garbage :)

Reply:
I suppose it's fine to use a suffix. Hard-drives are large these days :-).